torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -442,7 +450,15 @@ void fixDefaultRNNState(
   Node* unsqueezed_num_directions = graph->create(onnx::Unsqueeze, 1);
   unsqueezed_num_directions->insertBefore(n);
   unsqueezed_num_directions->addInput(num_directions->outputs()[0]);
-  unsqueezed_num_directions->is_(attr::axes, {0});
+  if (opset_version >= OPSET_VERSION_13) {
```

Comment:
nit: Would it be better that we create a new method with a meaningful name to increase readability and let people know why we have these new code? Like, change this line to "if (unsqueezeNeedsMultipleParameters(opset_version))". The method is simple as:

bool unsqueezeNeedsMultipleParameters(int opset_version) {
    return opset_version >= 13;
}

We can put it into helper.h file so this method could be shared among different places, like shape_type_inference.cpp and peehole.cpp.